app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsRelatedController.kt

```diff
@@ -22,12 +24,35 @@ class SettingsRelatedController : SettingsController() {
             defaultValue = false
             onClick {
                 if (isChecked) {
-                    RelatedJob.setupTask()
-                    RelatedJob.runTaskNow()
+                    RelatedUpdateJob.setupTask()
+                    RelatedUpdateJob.runTaskNow()
                 } else {
-                    RelatedJob.cancelTask()
+                    RelatedUpdateJob.cancelTask()
                 }
             }
         }
+
+        multiSelectListPreferenceMat(activity) {
+            key = Keys.relatedUpdateRestriction
+            titleRes = R.string.pref_related_update_restriction
+            entriesRes = arrayOf(R.string.wifi, R.string.charging)
+            entryValues = listOf("wifi", "ac")
+            customSummaryRes = R.string.pref_related_update_restriction_summary
+            onChange {
+                RelatedUpdateJob.setupTask()
+                true
+            }
+        }
+
+        preference {
+            title = "Github"
```

Comment:
I would change the title to Credits, and add a small description like
Credits 
For more information and to view the source code
link